SettingsView/Platforms/ios/Cells/CellBaseView.cs

```diff
@@ -405,7 +405,8 @@ protected virtual void UpdateIsEnabled()
         /// </summary>
         protected virtual void UpdateIsVisible()
         {
-            // If AccessoryView is set, hide the view because it overflows outside when IsVisible is false. 
+            ContentStack.Hidden = !CellBase.IsVisible;
```

Comment:
I got the issue that an EntryCell was visible even though IsVisible was set to false